lib/QBit/Application/Model/DB.pm

```diff
@@ -182,21 +182,24 @@ sub create_sql {
 
     my $meta = $self->get_all_meta();
 
-    $SQL .= join("\n\n", map {$self->$_->create_sql()} $self->_sorted_tables(keys(%{$meta->{'tables'}})))
+    $SQL .= join("\n\n",
+        map {$self->$_->create_sql()}
+          $self->_sorted_tables(grep {!@tables || in_array($_, \@tables)} keys(%{$meta->{'tables'}})))
```

Comment:
`in_array($_, \@tables)}` in cycle is N in square. Convert `@tables` to hash before cycle. 

It's not very important in current situation because count of tables cannot be huge, but use best practices :)